pylint/checkers/base/comparison_checker.py

```diff
@@ -76,8 +76,8 @@ class ComparisonChecker(_BasicChecker):
         "W0177": (
             "Comparison %s should be %s",
             "nan-comparison",
-            "Used when an expression is compared to NaN"
-            "values like numpy.NaN and float('nan')",
+            "Used when an expression is compared to NaN "
+            "values like numpy NaN and float('nan').",
```

Comment:
This is incorrect. `numpy.Nan` is the qualified name of the class.

Reply:
That's my fault, now corrected.